Corona-Warn-App/src/main/res/layout/view_country_list_entry_flag_item.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
`LinearLayout` with one child, could maybe be replaced with a `FrameLayout`?

Reply:
I concur!